codechecker_common/output/codeclimate.py

```diff
@@ -22,18 +22,27 @@ def convert(reports: List[Report]) -> Dict:
     """
     codeclimate_reports = []
     for report in reports:
-        codeclimate_reports.append(__to_codeclimate(report))
+        codeclimate_reports.append(__to_codeclimate(report, severity_map))
     return codeclimate_reports
 
+__codeclimate_severity_map = {
+    'UNDEFINED': 'info',
+    'LOW': 'minor',
+    'STYLE': 'minor',
+    'MEDIUM': 'high',
+    'HIGH': 'critical',
+}
 
-def __to_codeclimate(report: Report) -> Dict:
+def __to_codeclimate(report: Report, severity_map: Dict[str, str]) -> Dict:
     """Convert a Report to Code Climate format."""
     return {
         "type": "issue",
         "check_name": report.check_name,
         "description": report.description,
         "categories": ["Bug Risk"],
         "fingerprint": report.report_hash,
+        "severity": __codeclimate_severity_map.get(
```

Comment:
Some test cases are failed.
- https://github.com/Ericsson/codechecker/blob/master/analyzer/tests/functional/analyze_and_parse/test_analyze_and_parse.py#L310 - Missing severity level.
- Traceback (most recent call last):
  File "/home/runner/work/codechecker/codechecker/web/tests/functional/diff_local_remote/test_diff_local_remote.py", line 465, in test_diff_codeclimate_output
    self.assertTrue(os.path.exists(codeclimate_issues_file))

Could you please fix these test cases?

Reply:
fixed, though I couldn't run test_functional all the way. I don't have db set up for it.